.github/workflows/daily-qa.yml

```diff
@@ -28,19 +28,22 @@ jobs:
     outputs:
       latest-version: ${{ env.LATEST_VERSION }}
       previous-latest-version: ${{ env.PREVIOUS_LATEST_VERSION }}
+      previous-previous-latest-version: ${{ env.PREVIOUS_PREVIOUS_LATEST_VERSION }}
     steps:
       - uses: actions/checkout@v3
       - run: git fetch --tags
       - run: |
           latest=$(git tag | grep -E '^[0-9]+\.[0-9]+\.[0-9]+$' | sed -E -e 's/\.[0-9]+$//' | sort -Vr | head -n1)
           previous=$(git tag | grep -E '^[0-9]+\.[0-9]+\.[0-9]+$' | sed -E -e 's/\.[0-9]+$//' | grep -v "${latest}" | sort -Vr | head -n1)
-          if [ ! -z $latest ] && [ ! -z $previous ]; then
-            echo "Successfully computed latest versions: ${latest} and ${previous}"
+          previous_previous=$(git tag | grep -E '^[0-9]+\.[0-9]+\.[0-9]+$' | sed -E -e 's/\.[0-9]+$//' | grep -v "${latest}" | grep -v "${previous}" | sort -Vr | head -n1)
```

Comment:
:thought_balloon: I guess we should find a better way to do this one day, but I don't have anything OTOH so :shrug: